src/components/add-wine/validationSchema.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  wineName: {
+    required: true
+  },
+  wineType: {
+    required: true
+  },
+  wineYear: {
+    required: true,
+    validator: {
+      regEx: /^[0-9]{4}$/,
+      error: "Invalid Wine year"
+    }
+  },
+  wineCountry: {
+    required: true,
+    validator: {
+      regEx: /^[a-zA-Z\s]+$/,
+      error: "Invalid Wine country"
+    }
+  },
+  wineGrape: {
+    required: true,
+    validator: {
+      regEx: /^[a-zA-Z\s]+$/,
+      error: "Invalid Wine grape"
+    }
+  },
+  wineRegion: {
+    required: true,
+    validator: {
+      regEx: /^[a-zA-Z\s]+$/,
+      error: "Invalid Wine region"
+    }
+  },
+  sanderRating: {
+    required: true,
+    validator: {
+      regEx: /^([0-9]|10)$/,
```

Comment:
Ratings should allow decimals as well. But you don't need to worry. I can fix that when after I merge these changes :)

Reply:
Ooor you can fix it when you are fixing the other minor changes, if you want :D

Reply:
done